git-branchless/src/revset/grammar.rs

```diff
@@ -0,0 +1,2625 @@
+// auto-generated: "lalrpop 0.19.8"
```

Comment:
Did you try generating this at build time instead? Was it just too slow? For incremental builds, you can do something like https://github.com/martinvonz/jj/blob/6483aeefeaeb22462901ba0201b9b3a2f99052f4/lib/build.rs#L40-L43

Reply:
I got rid of the `build.rs` lately and didn't want to add it back so that I could keep build times down... but I might end up just adding it anyways, since it's a pain to manually regenerate the grammar each time.